google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -214,6 +214,15 @@ appengine.add.framework.support.choose.module.dialog.description={0} framework s
 appengine.add.framework.support.no.modules.message=All modules already have App Engine support, or no modules were found.
 appengine.add.framework.support.no.modules.title=Cannot add {0} framework support
 
+cloud.libraries.apireference.link=API Reference
+cloud.libraries.dialog.title=Add Cloud Libraries
+cloud.libraries.documentation.link=Documentation
+cloud.libraries.menu.action.text=Add Cloud libraries...
```

Comment:
Is this all that changed in the UI since the last commit (updated button text)? Maybe update the screenshot?

LGTM otherwise. approving.